src/clj/puppetlabs/services/master/master_service.clj

```diff
@@ -13,12 +13,96 @@
 
 (def master-service-status-version 1)
 
+;; Default list of allowed histograms/timers
+(def default-metrics-allowed-hists
+  ["compiler"
```

Comment:
I'm on the fence about that. Both arguments seem reasonable to me. I don't disagree that they should be close to the code that registers them, so if you think that's preferable I'm happy to move them there